apps/course/src/app/features/legal/legal-detail/legal-markdown-renderer/legal-markdown-renderer.component.html

```diff
@@ -0,0 +1,12 @@
+@if (isLoading) {
+  <app-loading-bars />
+} @else {
+  <div class="markdown-content">
+    <markdown
+      [src]="'assets/legal/content/' + legal?.file + '.md'"
+      lineNumbers
+      ngPreserveWhitespaces
```

Comment:
`lineNumbers` and `ngPreserveWhitespaces` can be removed as we don't have codeblocks in the legal file.